Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
--- a/src/Main.java
+++ b/src/Main.java	(date 1776784016146)
@@ -1,5 +1,8 @@
 public class Main {
     public static void main(String[] args) {
+        // TODO catalog - Сделать новый каталог для магазина, 1 поле - ID будет идти в формате int, 2 Title - в формате String,
+        //  3 Price - в формате double. Сделать полную инкапсуляцию и конструктор, в каталоге создаем 2 объекта(каталог1/2)
+
         // телефон
         MobileDevice phone = new MobileDevice("Xiaomi Galaxy S200 Ultra", 190000, "Смартфон "
                 + "созданный совместно с компаниями Xiaomi, Samsung и Vivo, внутрь встроен новый процессор "
@@ -13,35 +16,21 @@
         Electronic lamp = new Electronic("Настольная лампа", 3000, "Светодиодная лампа Xiaomi "
                 + "Leomax Ultra, яркость этой лампы позволит увидеть ближайшие 5 городов!");
 
-        // добавляем в список продуктов
-        Product.addProduct(phone);
-        Product.addProduct(shovel);
-        Product.addProduct(lamp);
+        // Добавляем в общий список
+        Catalog.addCategory(phone);
+        Catalog.addCategory(shovel);
+        Catalog.addCategory(lamp);
 
-        // добавляем базовые категории
-        Catalog.addCategory(new Category("Электроника", 0, "Электро-товары") {
-            @Override
-            public void showInfo() {
-                System.out.println("[Категория] " + getTitle() + ", [Описание] " + getDescription() + " (ID: " + getID() + ")");
-            }
-        });
-        Catalog.addCategory(new Category("Дача и сад", 0, "Товары для дома и дачи") {
-            @Override
-            public void showInfo() {
-                System.out.println("[Категория] " + getTitle() + ", [Описание] " + getDescription() + " (ID: " + getID() + ")");
-            }
-        });
+        // Добавляем базовые категории
+        Catalog.addCategory(new Category("Электроника", 0, "Электро-товары"));
+        Catalog.addCategory(new Category("Дача и сад", 0, "Товары для дома и дачи"));
+
 
         // вывод категорий и товаров
         System.out.println("Список категорий и товаров:");
         Catalog.showCategories();
-        Product.getProductList();
     }
 }
-
-// TODO - этап 1:
-//  1.1 - Сделать новый каталог для магазина, 1 поле - ID будет идти в формате int, 2 Title - в формате String,
-//  1.2 - 3 Price - в формате double. Сделать полную инкапсуляцию и конструктор, в каталоге создаем 2 объекта(каталог1/2)
 
 // TODO - этап 2:
 //  2.1 - в классе Product (добавить) должно быть поле Description - описание;
@@ -55,9 +44,4 @@
 
 // TODO - этап 3:
 //  3.1 - нужно создать класс Category (уже есть), добавить ADD_Category, SHOW_Category
-//  3.2 - нужно создать счетчик в Category: 1 - сколько категорий, 2 - сколько sub категорий
-
-// TODO - этап 4 (полиморфизм)
-//  4.1 - надо в Category добавить абстрактный метод Show info, который мы будем переобразовывать
-//  4.2 - создать несколько разных товаров в Product в списке в виде Array list
-//  4.3 - надо в show info что бы весть список показать на экране
\ No newline at end of file
+//  3.2 - нужно создать счетчик в Category: 1 - сколько категорий, 2 - сколько sub категорий
\ No newline at end of file
